src/components/inputCheckbox/AposCheckbox.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <label class="apos-choice-label" :for="id">
+    <input type="checkbox" class="apos-sr-only apos-input--choice apos-input--checkbox"
+      :value="choice.value" :name="field.name" :id="id" :aria-label="choice.label"
+      tabindex="0" :disabled="status.disabled" @input.stop="toggle" v-model="value.data"
+    />
+    <span class="apos-input-indicator" aria-hidden="true">
+      <component :is="`${
+          field.indeterminate ? 'MinusIcon' : 'CheckBoldIcon'
+        }`"
+        :size="10" v-if="value.data.includes(choice.value)"></component>
+    </span>
+    <span :class="{'apos-sr-only': field.hideLabel }" v-if="choice.label" class="apos-choice-label-text">{{ choice.label }}</span>
+  </label>
+</template>
+
+<script>
+import CheckBoldIcon from "vue-material-design-icons/CheckBold.vue";
+import MinusIcon from "vue-material-design-icons/Minus.vue";
+
+export default {
+  components: {
+    CheckBoldIcon,
+    MinusIcon
+  },
+  props: {
+    choice: Object,
+    field: Object,
+    value: Object,
+    status: Object,
+    id: String,
+  },
+  methods: {
+    toggle() {
+      this.$emit('toggle', this.value);
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+  @import '../../scss/_inputs';
```

Comment:
Technically not necessary since this is imported an unscoped in `AposInputWrapper`. There are decent arguments for both scoping it there and not, but right now it's set up so you don't need to import this again.